src/raven_api/indicators.py

```diff
@@ -312,50 +329,655 @@ def annual_peaks(
         SELECT site, water_year, MAX(value) AS annual_peak
         FROM parquet_scan('{parquet_path}')
         WHERE value IS NOT NULL
-        {site_filter}
+        {sites_filter}
         {date_filter}
         GROUP BY site, water_year
         ORDER BY site, water_year
         """
     return con.execute(query).fetchdf()
 
 
+def detect_outliers_duckdb(

```

Comment:
We should create a `raven_api/utils.py` file to organize things in order to isolates indicator functions. Functions like this (and `calculate_goodness_of_fit`, `fit_distribution`, `select_best_distribution`, etc...) that are used to help can be moved there and imported here.